File: Prática/Lista 01 - Java /Ex07/src/ex07/Ex07.java
package ex07;

import java.util.Scanner;

/**
 * @author bryan
 */
public class Ex07 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int quantidadeVeiculos = 5;
        double[] velocidades = new double[quantidadeVeiculos];
        int quantidadeMultas = 0;
        double valorTotalArrecadado = 0;

        for (int i = 0; i < quantidadeVeiculos; i++) {

            System.out.printf("Digite a velocidade do %dº veículo: ", (i + 1));
            velocidades[i] = sc.nextDouble();
        }

        sc.close();

        for (int i = 0; i < velocidades.length; i++) {

            if (velocidades[i] > 60) {
                quantidadeMultas++;
                valorTotalArrecadado += 150;
            }
        }

        System.out.println("\nA quantidade de veículos multados foi: " + quantidadeMultas);
        System.out.printf("Com um valor total arrecadado de R$ %.2f \n", valorTotalArrecadado);

    }

}

File: Prática/Lista 04 - C/Ex07/Ex07.c
#include "../Console.c"
#include <stdio.h>
#include <stdlib.h>

main(){
    int quantVeiculos = 5;
    int quantMultas = 0;
    int velocidadeMax = 60;
    float totalArrecadado = 0;
    float velocidade;

    for(int i = 0; i < quantVeiculos; i++){
        clear();
        printf("Digite a velocidade do %dº veículo: ", i+1);
        scanf("%f", &velocidade);  
        if(velocidade > velocidadeMax){
            quantMultas++;
            totalArrecadado += 150;
        }
    }
    
    clear();

    printf("%d veículos foram multados, tendo um total arrecadado de R$ %.2f\n\n", quantMultas, totalArrecadado);
    
}